app/system/helpers/support_helper.php

```diff
@@ -161,7 +161,7 @@ function mdate($format = null, $time = null)
      *
      * @return int
      */
-    function convert_php_to_moment_js_format($format)
+    function mdate_to_moment_js_format($format)
```

Comment:
Funny how I missed this.. I actually opened the file to get the correct function name. Anywhos, the name should be `convert_php_to_moment_js_format` as its being used here https://github.com/tastyigniter/TastyIgniter/blob/cfdc38cf3aaabf254326a4912b115a964de4cfb2/app/admin/formwidgets/DatePicker.php#L134

Reply:
:) Updated